src/redux/actions/concepts/dictionaryConcepts.js

```diff
@@ -436,9 +436,8 @@ export const fetchExistingConcept = conceptUrl => async (dispatch) => {
     dispatch(prepopulateAnswers(answers));
     dispatch(prePopulateSets(sets));
   } catch (error) {
-    notify.show('Could not retrieve concept details', 'error', 3000);
-    if (error.response) {
-      dispatch(isErrored(error.response.data, FETCH_EXISTING_CONCEPT_ERROR));
```

Comment:
Is there any reason we can't also get rid of `FETCH_EXISTING_CONCEPT_ERROR` since it is not really helping with anything?

In the reducer;
```
case FETCH_EXISTING_CONCEPT_ERROR:
      return {
        ...state,
      };
```

Reply:
I have removed it since its nolonger needed.